release/_portal_release_tester.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+function get_githash {
+    local branch_name=${1}
+
+	if [[ "${branch_name}" =~ "7.4." ]]
+	then
+		local curl_url="https://api.github.com/repos/brianchandotcom/liferay-portal/commits/${branch_name}"
+	else
+		local curl_url="https://api.github.com/repos/brianchandotcom/liferay-portal-ee/commits/${branch_name}"
+	fi
+
+	local curl_response=$(\
+				curl \
+					${curl_url} \
+					-H "Accept: application/vnd.github.VERSION.sha" \
+					-H "Authorization: Bearer ${GH_TOKEN}" \
+					-s)
+
+    echo "${curl_response}"
```

Comment:
@joshitagaki, there is a file named [_github.sh](https://github.com/liferay/liferay-docker/blob/master/release/_github.sh) which aims to hold all GitHub API code, can you please move this function to be there?